src/kOS.Safe/Execution/CPU.cs

```diff
@@ -1052,8 +1095,6 @@ public void KOSFixedUpdate(double deltaTime)
             instructionsSoFarInUpdate = 0;
             var numMainlineInstructions = 0;
 
-            firstInstructionPointerInUpdate = InstructionPointer;
-
```

Comment:
This was never accurate anyway as if you had a trigger that lasted longer than 1 update, on the second update's worth of running it, this would mark an instruction in the middle of the trigger, and the system would never go back to mainline mode until it hit this instruction, which it falsely believed to be the mainline instruction it had first branched into the triggers from (it was in the first update the trigger fired in, but if it's a long trigger lasting more than 1 update, then in the second update, it's no longer pointing at the mainline code and that meant it would fail to realize it hit mainline code for at least one whole extra update after it already had.)